docs/source-pytorch/accelerators/hpu_intermediate.rst

```diff
@@ -66,3 +66,34 @@ This enables advanced users to provide their own BF16 and FP32 operator list ins
     trainer.fit(model, datamodule=dm)
 
 For more details, please refer to `PyTorch Mixed Precision Training on Gaudi <https://docs.habana.ai/en/latest/PyTorch_User_Guide/PyTorch_User_Guide.html#pytorch-mixed-precision-training-on-gaudi>`__.
+
+----
+
+Enabling DeviceStatsMonitor with HPUs
```

Comment:
let's make sure this get's merged on master after the release.

Reply:
@rohitgr7 What? We want it for the release.

Reply:
@kaushikb11 
did you check with @carmocca !

I suggested this for v1.7 last week, but due to feature freeze it couldn't make it.

check the milestone.

Reply:
@Borda What should we do about it? It's not a major feature but rather an enhancement to align the updates with the Habana API.

Reply:
If it is stable as it seems so, let's add it...